Makefile

```diff
@@ -262,6 +288,46 @@ else
 	$(error addlicense must be installed for this rule: go get -u github.com/google/addlicense)
 endif
 
+### gen_terminal_csv : generate the csv for a newer version
+gen_terminal_csv : update_devworkspace_crds update_terminal_manifests
+	operator-sdk generate csv --apis-dir ./pkg/apis --csv-version 1.0.0 --make-manifests --update-crds --operator-name "web-terminal" --output-dir ./web-terminal-operator
+	
+	# filter the deployments so that only the valid deployment is available. See: https://github.com/eclipse/che/issues/17010
+	cat ./web-terminal-operator/manifests/web-terminal.clusterserviceversion.yaml | \
+	yq -Y \
+	'.spec.install.spec.deployments[] |= select( .spec.selector.matchLabels.app? and .spec.selector.matchLabels.app=="che-workspace-controller")' | \
+	tee ./web-terminal-operator/manifests/web\ terminal.clusterserviceversion.yaml >>/dev/null
+
+	cp devworkspace-crds/deploy/crds/workspace.devfile.io_devworkspaces_crd.yaml web-terminal-operator/manifests
+
+### olm_build_bundle: build the bundle and push it to a docker registry
+olm_build_bundle: _print_vars
```

Comment:
does it make sense to give it terminal-specific name as well?

Reply:
Probably yes. I'll rename all the web terminal specific OLM stuff to something like olm_build_terminal_bundle etc